styles.css

```diff
@@ -63,3 +63,40 @@ form {
   font-size: 24pt
 }
 
+.fa-arrow-circle-down,
+.fa-arrow-circle-right {
+  margin-top: 20px;
+  font-size: 24pt;
+}
+
+span {
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+  font-size: 14pt;
+  margin-left: 20px;
+  font-weight: 400
+}
+
+#practice-answer {
+  margin-top: 20px
+}
+
+.fa-angle-left,
+.fa-angle-right {
+  font-size: 54pt;
+  /* margin: 50px 200px 0px; */
+  color: rgb(52, 58, 64)
+}
+
+.controls {
+  margin: 0 auto;
+  margin-top: 50px;
+  max-width: 350px
+}
+
+.fa-angle-right {
+  float: right
+}
+
+.fa-angle-left {
+  float: left
```

Comment:
Avoid augmenting classes defined in libraries. Define your own classes and add them.